### DISCUSSION TITLE
Broken start screen when changing code

### DISCUSSION THREAD

**Discussion by Ari-Sillycore** [KadeArchive/Kade-Engine#721] [Q&A]
Hey kade so I need to tell you that changing the start screen on the new verison of Kade engine breaks the start screen. Do you know why this happens? Thank you!
![broken](https://user-images.githubusercontent.com/82355148/120903090-01a43780-c5f9-11eb-9279-78e9ae7dcfb2.png)

**Comment by ACrazyTown** [comment]
What exactly did you do
